plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetModel.java

```diff
@@ -707,6 +707,33 @@ public void setData(@NotNull List<Object[]> rows) {
         hasData = true;
     }
 
+    private void processColorOverrides(DBVEntity virtualEntity) {
+        List<DBVColorOverride> coList = virtualEntity.getColorOverrides();
+        if (!CommonUtils.isEmpty(coList)) {
+            for (DBVColorOverride co : coList) {
+                DBDAttributeBinding binding = DBUtils.findObject(attributes, co.getAttributeName());
+                if (binding != null) {
+                    List<AttributeColorSettings> cmList =
+                            colorMapping.computeIfAbsent(binding, k -> new ArrayList<>());
+                    cmList.add(new AttributeColorSettings(co));
+                } else {
+                    log.debug("Attribute '" + co.getAttributeName() + "' not found in bindings. Skip colors.");
+                }
+            }
+        }
+    }
+
+    public void updateColorMapping(DBVEntity virtualEntity, boolean reset) {
+        colorMapping.clear();
+        if (virtualEntity == null) {
+            return;
+        }
```

Comment:
Redundant because `virtualEntity` is never `null` according to the call site

Reply:
Done!

Reply:
Add the `@NotNull` annotation as well please

Reply:
Already added

Reply:
Pardon, in the method above.